boards/risc-v/mpfs/icicle/scripts/ld-envm.script

```diff
@@ -27,6 +27,9 @@ MEMORY
 
 OUTPUT_ARCH("riscv")
 
+__ksram_start = ORIGIN(lim);
+__ksram_size = LENGTH(lim);
```

Comment:
Why not to define `__ksram_end = ORIGIN(lim) + LENGTH(lim)` and use it instead of `((uintptr_t)&__ksram_start + (uintptr_t)&__ksram_size)`?

Reply:
Sure